freeipa/src/main/resources/freeipa-salt/salt/freeipa/common-install.sls

```diff
@@ -109,21 +109,3 @@ restart_sssd_if_reconfigured:
     - failhard: True
     - watch:
       - file: /etc/sssd/sssd.conf
-
-/opt/salt/scripts/disable_anon_ldap.sh:
```

Comment:
just to be sure. This works when customer upgradeing freeipa right ?

Reply:
during freeipa upgrade salt states are upgraded, so yes, everything should be fine